data/mods/Magiclysm/Spells/earthshaper.json

```diff
@@ -66,5 +66,91 @@
     "energy_increment": 75,
     "final_energy_cost": 1000,
     "difficulty": 5
+  },
+  {
+    "id": "eshaper_shardspray",
+    "type": "SPELL",
+    "name": "Shardspray",
+    "description": "This spell projects a wide spray of sharp metal shards, cutting into your foes and friends alike.",
+    "valid_targets": [ "hostile", "ground", "ally" ],
+    "effect": "cone_attack",
+    "spell_class": "EARTHSHAPER",
+    "energy_source": "MANA",
+    "difficulty": 3,
+    "base_casting_time": 100,
+    "base_energy_cost": 100,
+    "max_level": 20,
+    "min_damage": 10,
+    "max_damage": 60,
+    "damage_increment": 2,
+    "damage_type": "cut",
+    "min_aoe": 25,
+    "max_aoe": 60,
+    "aoe_increment": 5,
+    "min_range": 1,
+    "max_range": 10,
+    "range_increment": 0.5
+  },
+  {
+    "id": "eshaper_piercing_bolt",
+    "type": "SPELL",
+    "name": "Piercing Bolt",
+    "description": "This spell projects a piercing rod of conjured iron at those that dare oppose you.",
+    "valid_targets": [ "hostile" ],
+    "effect": "projectile_attack",
+    "spell_class": "EARTHSHAPER",
+    "energy_source": "MANA",
+    "base_casting_time": 100,
+    "base_energy_cost": 100,
```

Comment:
this should be much higher, since this spell is so strong and not even a high difficulty.

Reply:
My thinking was that since the established base attack spell was type none and penetrated obstacles and could do up to 20 points of damage for a cost of 50 mana, anything that does not benefit from those things should count them as balancing factors.
This does stab damage and is a projectile, not a targeted attack that nothing can resist. There are creatures that this should not be able to hurt and it's not a versatile or useful as a magic missile on those grounds, so I gave it a higher range and damage up to the shotgun area when maxed out. they still have to practice the spell a lot to reach those numbers, and probably a lot more than MM.
If you don't find that persuasive, does 115-125 sound reasonable?